tests/fast-integration/eventing-test/metric-test.js

```diff
@@ -23,7 +23,8 @@ const dashboards = {
     // https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries
     assert: function (result) {
       expect(result.length).to.be.greaterThan(0, "No value found in the result");
-      expect(getMetricValue(result[0])).to.be.greaterThan(0);
```

Comment:
This was breaking the whole fi tests, because sometimes we were receiving the metric, but the value of the metric was 0, the same happened here: https://github.com/kyma-project/kyma/pull/12882